ot/sliced.py

```diff
@@ -0,0 +1,100 @@
+"""
+Sliced Wasserstein Distance.
+
+"""
+
+# Author: Adrien Corenflos <adrien.corenflos@gmail.com>
+#
+# License: MIT License
+
+
+import numpy as np
+
+
+def _random_projections(n_projections, dimension, random_state):
```

Comment:
`get_random_projection` ? 

Also this function can be used elsewhere I would not hide it and it needs a more detailled doc